packages/main/pyproject.toml

```diff
@@ -1,6 +1,6 @@
 [tool.poetry]
 name = "rpaframework"
-version = "11.1.3"
+version = "11.2.0"
```

Comment:
I'll need the rights for publishing in PyPI for both `rpaframework` and `rpaframework-dialogs` I think.